README.hu.md

```
@@ -0,0 +1,502 @@
+5. Click the **Authorize** button.
+   ![step4](docs/images/openrouter/4.png)
+
+6. Prepay as much as you plan to use.
+   ![step5](docs/images/openrouter/5.png)
+
+<details>
+<summary><h3>Ha az Authorize gomb nem működött</h3></summary>
+
+Ha az Authorize gombra kattintott, de még mindig nincs hitelesítve, próbálja újra, vagy hozzon létre API kulcsot kézzel:
+
+6. Click your account in the top right, go to the API Keys tab on the left, then click the Create button in the center.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)